### PR TITLE
chore(inventory): Make sure `mender_client_version` is always submitted.

### PR DESCRIPTION
**Pull request by kacf**
Ticket: MEN-6840

### CODE REVIEW

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Remember to remove it from [here](https://github.com/mendersoftware/mender/blob/7a793e54e955dfc68cfb466c32526b9c3e3fb630/support/mender-inventory-provides#L11).

Otherwise it looks good! Please run the integration tests again after removing the mentioned lines in the script to make sure the relevant tests still pass :+1:

File: src/mender-update/inventory_test.cpp [hunk 1]
```diff
@@ -96,7 +99,8 @@ exit 0
 	NoAuthHTTPClient client {client_config, loop};
 
 	const string expected_request_data =
-		R"([{"name":"key1","value":["value1","value11"]},{"name":"key2","value":"value2"},{"name":"key3","value":"value3"}])";
+		R"([{"name":"key1","value":["value1","value11"]},{"name":"key2","value":"value2"},{"name":"key3","value":"value3"},{"name":"mender_client_version","value":["additional_version",")"
```
Comment (oleorhagen) [inline src/mender-update/inventory_test.cpp:102]: Why are we submitting multiple? :thinking:

Comment (kacf) [reply src/mender-update/inventory_test.cpp:102]: Honestly I don't really have a usecase for this. But it was the only way I could think of which doesn't silently hide the existing value if it has also been provided by a script. Most likely if you end up in this situation, you've done something wrong, but this way you will at least be able to see that.

Comment (kacf) [reply src/mender-update/inventory_test.cpp:102]: It's still redundant. I don't think it should be doing it at all.

If two different scripts provided the value, then we'd also make an array, so this is consistent with that.

Comment (oleorhagen) [reply src/mender-update/inventory_test.cpp:102]: :+1:  Alright. Let's hope we don't mess something up :pinched_fingers:

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**